src/app/api/course/route.test.ts

```diff
@@ -34,15 +36,28 @@ describe('API', () => {
 
   describe('POST', () => {
     it('adds new course in to the database', async () => {
-      const response = await POST(newCourse as any);
+      const { req } = createMocks<NextRequest>({
+        method: 'POST',
+        body: newCourse,
+        json: () => newCourse,
```

Comment:
Body & json both required?

Reply:
Noup. Just the json function used. Body left there for some unknown reason 🤷‍♂️